pkg/services/reputation/eigentrust/calculator/deps.go

```diff
@@ -59,3 +59,9 @@ type IntermediateWriter interface {
 type IntermediateWriterProvider interface {
 	InitIntermediateWriter(Context) (IntermediateWriter, error)
 }
+
+// AlphaProvider must provide information about required
+// alpha parameter for eigen trust algorithm.
+type AlphaProvider interface {
+	EigenTrustAlpha() (float64, error)
```

Comment:
There are no other alphas, it can be just `Value` or `Get`.

Reply:
in fact i wrote it in such way, cause i was almost sure that we would be glad to store `Alpha` in global config so i named the method as it is used to be named in global config getters(e.g. `EigenTrustIterations`)